src/category_theory/differential_object.lean

```diff
@@ -104,6 +104,41 @@ lemma zero_f (P Q : differential_object C) : (0 : P ⟶ Q).f = 0 := rfl
 
 end differential_object
 
+namespace functor
+
+universes v' u'
+variables (D : Type u') [category.{v'} D]
```

Comment:
Was it intentional to change `D` from implicit to explicit? cf. https://github.com/leanprover-community/lean-liquid/blob/648a68380264e2d7aa65d902143f6f4d125b2251/src/for_mathlib/category_theory.lean#L16